xml/schema/Contact/SavedSearch.xml

```diff
@@ -18,6 +18,39 @@
     <name>id</name>
     <autoincrement>false</autoincrement>
   </primaryKey>
+
+  <field>
+    <name>name</name>
+    <title>Saved Search Name</title>
+    <type>varchar</type>
+    <length>255</length>
+    <default>NULL</default>
+    <comment>Unique name of saved search</comment>
+    <html>
+      <type>Text</type>
+    </html>
+    <add>1.0</add>
+  </field>
+  <index>
+    <name>UI_name</name>
+    <fieldName>name</fieldName>
+    <unique>true</unique>
+    <add>5.32</add>
+  </index>
+
+  <field>
+    <name>label</name>
```

Comment:
Does this need "localizable"?

Reply:
No I don't think so. This is intended for back-end administration screens.